src/licensedcode/plugin_license.py

```diff
@@ -176,6 +187,7 @@ def get_scanner(
         license_text_diagnostics=False,
         license_url_template=SCANCODE_LICENSEDB_URL,
         unknown_licenses=False,
+        additional_directories=None,
```

Comment:
I am not sure this would be needed further down. Instead the license index has these details